src/app/api/v1/user/breaches/bulk-resolve/route.ts

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextRequest, NextResponse } from "next/server";
+import { authOptions } from "../../../../utils/auth";
+import { getServerSession } from "next-auth";
+import { logger } from "../../../../../functions/server/logging";
+import {
+  BreachBulkResolutionRequest,
+  Subscriber,
+} from "../../../../../(nextjs_migration)/(authenticated)/user/breaches/breaches.js";
+import { getBreaches } from "../../../../../functions/server/getBreaches";
+import { getAllEmailsAndBreaches } from "../../../../../../utils/breaches";
+import {
+  getSubscriberByEmail,
+  setBreachResolution,
+} from "../../../../../../db/tables/subscribers";
+
+export async function PUT(req: NextRequest): Promise<NextResponse> {
+  const session = await getServerSession(authOptions);
+  if (!session?.user?.subscriber || typeof session.user?.email !== "string") {
```

Comment:
Should this be `session?.user?.email` to match `session?.user?.subscriber`?
Not sure how likely `session` is to be undefined.

Reply:
I’ll go ahead and merge this to unblock another PR, but will add this and the aforementioned change there! :)